pkgs/top-level/all-packages.nix

```diff
@@ -121,7 +121,7 @@ with pkgs;
 
   dieHook = makeSetupHook {} ../build-support/setup-hooks/die.sh;
 
-  dockerTools = callPackage ../build-support/docker { };
+  dockerTools = callPackage ../build-support/docker { go = go_1_9; };
```

Comment:
Can you also downgrade dockers go version and add this link as reference to both? https://github.com/moby/moby/pull/35739